src/backend/TrafficCourts/Test/Arc.Dispute.Service/Mappings/MappingTests.cs

```diff
@@ -63,5 +63,22 @@ public void Test_convert_data_from_TcoDisputeTicket_to_ArcFileRecord_list()
                 }
             }  
         }
+
+        [Fact]
+        public void can_parse_full_section_with_valid_adnotated_ticket_and_full_section()
```

Comment:
I might recommend you convert this test into a [Theory] (a parametrized test) and try to parse out several of the known fullsection types we see in the real world.  The full list is [here](https://github.com/bcgov/jag-traffic-courts-online/blob/develop/src/backend/oracle-data-api/src/main/resources/data/statutes.csv).

If I put that list into Excel and sort by the section, some examples are:
24.1(1)(b.1)(viii)
24(4) Section (3)(a)
1271271271(2222)(bbb)(iiiii)
2(2)(a)(i)
2(1)(g)
2(9)
4

I'm curious how the second one will work in this code.